/social/process/how/how-v1 (#1)
holochain/how: HOW/social/process/_requirements.md

---

> Status: <DRAFT|REVIEW|REJECTED|OBSOLETE|DEFERRED|ALIVE>

Reviewer: ```suggestion
Status: <DRAFT|REVIEW|REJECTED|OBSOLETE|DEFERRED|LIVE>
```

nitpick but "alive" feels odd when we're not using any other lifecycle terms like "dead", etc.

> Status: <DRAFT|REVIEW|REJECTED|OBSOLETE|DEFERRED|ALIVE>

Reviewer: ```suggestion
Status: <DRAFT|REVIEW|REJECTED|DEFERRED|ALIVE|SUPERSEDED>
```

nitpick but feels more clear to me

Author: I like it.

---
## Process Specification

The process specification MUST describe the workflow of the process, including the stages, steps and statuses the process goes through, and transition events that trigger status and stage changes.

## Process Template Section: REVIEW

After a standard is drafted, when it moves into the `REVIEW` status the following section must be added:

### Steward

Person or Working Group who is stewarding this HOW and is responsible for moving it through the workflow as specified by the process.

## Process Template Section: REJECTED, OBSOLETE, DEFERRED

When a standard is not accepted or is obsoleted the following section must be added:

### <status> Reason

The reason this standard was not accepted or was obsoleted.  In the case of obsoleting the content of this section MUST include which HOW, or HOW version makes this HOW obsolete.

## Process Template Section: ALIVE

> When a standard is aligned on an is set to status ALIVE this section must be added:

Reviewer: ```suggestion
When a standard is set to status ALIVE, all previous versions of the same standard must be set to status SUPERSEDED.

When a standard set to status ALIVE this section must be added:
```
